shells/pdksh/Portfile

```diff
@@ -14,13 +16,14 @@ long_description	\
 	initially created by Eric Gisin using Charles Forsyth's public \
 	domain V7 shell as a base as well as parts of the BRL shell. \
 	Since then a number of people have maintained and contributed to it
+
 platforms	darwin
-homepage	http://www.cs.mun.ca/~michael/pdksh/
-master_sites	ftp://ftp.cs.mun.ca/pub/pdksh/ \
-		ftp://gd.tuwien.ac.at/utils/shells/pdk \
-		ftp://ftp.lip6.fr/pub/unix/shells/pdksh/ \
-		ftp://ftp.rge.com/pub/shells/pdksh/
-checksums	md5 871106b3bd937e1afba9f2ef7c43aef3
+homepage    ftp://ftp.lip6.fr/pub/unix/shells/pdksh/
```

Comment:
The homepage cannot be an ftp URL. Please change it to something with http(s).

Reply:
What do you suggest? I can not  find one via Google.

Reply:
We usually use the last version of the page recorded at the Internet Archive. In this case, that's:

https://web.archive.org/web/20160918190548/http://www.cs.mun.ca/~michael/pdksh/